src/FqnUtilities.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace LanguageServer;
+
+use phpDocumentor\Reflection\{Type, Types};
+use Microsoft\PhpParser;
+
+class FqnUtilities
+{
+    /**
+     * Returns all possible FQNs in a type
+     *
+     * @param Type|null $type
+     * @return string[]
+     */
+    public static function getFqnsFromType($type): array
```

Comment:
Is there any benefit in having this as a class with static methods instead of pure functions?